scripts/prepare-machine.ps1

```diff
@@ -117,28 +114,6 @@ if ($InitSubmodules) {
     }
 }
 
-function Install-ClogTool {
```

Comment:
You'll notice the `if ($InitSubmodules)` section above. It will need to be updated to `init` the clog submodule too.